test/gtest/source-map.cpp

```diff
@@ -20,32 +20,136 @@
 
 using namespace wasm;
 
-using SourceMapTest = PrintTest;
+class SourceMapTest : public PrintTest {
+  std::vector<char> buffer;
 
-// Check that debug location parsers can handle single-segment mappings.
-TEST_F(SourceMapTest, SourceMappingSingleSegment) {
-  auto text = "(module)";
+protected:
   Module wasm;
-  parseWast(wasm, text);
+  std::unique_ptr<SourceMapReader> reader;
+
+  void SetUp() override {
+    PrintTest::SetUp();
+    reader.reset();
+    parseWast(wasm, "(module)");
+  }
 
+  void ParseMap(std::string& sourceMap) {
```

Comment:
```suggestion
  void parseMap(std::string& sourceMap) {
```
This is inconsistent with the `SetUp` from gtest itself, but more consistent with our usual conventions. I don't feel strongly though.